src/main/java/com/kiwigrid/helm/maven/plugin/UploadMojo.java

```diff
@@ -154,6 +195,33 @@ protected HttpURLConnection getConnectionForUploadToNexus(File file) throws IOEx
 		return connection;
 	}
 
+	protected void uploadToHarbor(File file) throws IOException, BadUploadException, MojoExecutionException {
```

Comment:
Upload to Harbor is implemented different from other uploads.

I would prefer to get a method like
```
protected HttpURLConnection getConnectionForUploadToHarbor() throws IOException {
}
```

What do you think?